README.md

```diff
@@ -1 +1,65 @@
-# Bookstore
\ No newline at end of file
+# Bookstore
+
+## Description
+
+This project is used to display a list of books. Users can add/remove books from the list. It is built with ReactJS and Redux. It implements Routing, and Components concepts
+
+## Screenshot
+
+![Coming Soon](https://user-images.githubusercontent.com/)
+
```

Comment:
- [ ] The link to the screenshot is broken, please kindly fix that.

Reply:
done!